models/dashboard_subsidio_sppo/sumario_servico_tipo_viagem_dia.sql

```diff
@@ -19,7 +19,7 @@ WITH
     viagens_subsidio AS viagens,
     ROUND(distancia_total_subsidio, 2) AS km_apurada
   FROM
-    `rj-smtr`.`dashboard_subsidio_sppo`.`sumario_dia`
+     `rj-smtr`.`dashboard_subsidio_sppo`.`sumario_dia`
```

Comment:
```suggestion
     {{ ref("sumario_servico_dia") }}
```

É para ser `sumario_servico_dia` ne? Essa tabela `sumario_dia` não tá versionada